### PR TITLE
Update kubelet image with fix to inaccurate pod CPU usage report 

### PR DESCRIPTION
**Pull request by xuto2**
https://github.com/kubernetes/kubernetes/pull/83507 fix on  	1.15.11	1.15.12	1.16.10	1.16.13	1.17.7	1.17.9	
https://github.com/kubernetes/kubernetes/pull/82300 fix on  	1.15.11	1.15.12	1.16.10	1.16.13

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -402,27 +402,27 @@ done
 # need to cover previously supported version for VMAS scale up scenario
 # So keeping as many versions as we can - those unsupported version can be removed when we don't have enough space
 # below are the required to support versions
-# v1.15.11-hotfix.20200714.1
-# v1.15.12-hotfix.20200714.1
-# v1.16.10-hotfix.20200714.1
-# v1.16.13-hotfix.20200714.1
-# v1.17.7-hotfix.20200714.2
-# v1.17.9-hotfix.20200714.1
+# v1.15.11-hotfix.20200817.1
```
Comment (AbelHu) [inline vhdbuilder/packer/install-dependencies.sh:405]: Windows does not patch 1.15.11 and 1.15.12 because they will be retired soon.

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:405]: I know. I'm doing this as it's a free lunch to me.

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -471,17 +471,23 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.10-hotfix.20200408.1
 1.15.11-hotfix.20200529.1
 1.15.11-hotfix.20200714.1
+1.15.11-hotfix.20200817.1
```
Comment (bowang-666) [inline vhdbuilder/packer/install-dependencies.sh:474]: idk how much additional space this'll take, but we shall see when build vhd

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:474]: ==> azure-arm: + df -h
    azure-arm: Filesystem      Size  Used Avail Use% Mounted on
    azure-arm: udev            3.4G     0  3.4G   0% /dev
    azure-arm: tmpfs           695M  8.7M  687M   2% /run
    azure-arm: /dev/sda1        29G   27G  2.2G  93% /
    azure-arm: tmpfs           3.4G     0  3.4G   0% /dev/shm

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (AbelHu) [reply vhdbuilder/packer/install-dependencies.sh:405]: We should follow the same policy for both Windows and Linux. If we pick these changes for Linux, we should also pick these for Windows.

File: vhdbuilder/packer/install-dependencies.sh [hunk 3]
```diff
@@ -471,17 +471,23 @@ PATCHED_HYPERKUBE_IMAGES="
...
 1.16.10-hotfix.20200623.1
 1.16.10-hotfix.20200714.1
+1.16.10-hotfix.20200817.1
 1.16.13-hotfix.20200714.1
+1.16.13-hotfix.20200817.1
 1.17.3-hotfix.20200601.1
 1.17.7-hotfix.20200624
 1.17.7-hotfix.20200714.1
 1.17.7-hotfix.20200714.2
+1.17.7-hotfix.20200817.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:488]: this where we introduce this.

**Pull request merged**